backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/utils/MutableBoundStatementWrapper.java

```diff
@@ -0,0 +1,38 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.backends.cassandra.utils;
+
+import com.datastax.oss.driver.api.core.cql.BoundStatement;
+
+public class MutableBoundStatementWrapper {
```

Comment:
What do you think about it?
https://github.com/chibenwa/james-project/pull/35/files#diff-5cb9b0977b0b982019606db36c7580461dbd8b1b621e317a4adeecb29cf1adf2

I use it for BoundStatementBuilder